README.md

```diff
@@ -1,4 +1,6 @@
-# `@shopify/koa-shopify-auth`
+# DEPRECATED `@shopify/koa-shopify-auth`
+
+NOTE: this repo is no longer maintained. See [simple-koa-shopify-auth](https://github.com/TheSecurityDev/simple-koa-shopify-auth) for a potential community solution.
```

Comment:
We should probably also direct folks to https://github.com/Shopify/shopify-api-node as the new official solution here!

Reply:
good idea